tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -130,6 +80,24 @@ public function testEventPage(): void {
       trim($content_banner->filter('.oe-sc-event__oe-summary')->text())
     );
 
+    $date = $crawler->filter('dl > dd');
+
+    // Assert event dates starting and ending same day.
+    $this->assertEquals('Wednesday 09 February 2022, 21.00-23.00 (CET)', trim($date->text()));
+
+    // Assert event dates starting and ending at different days.
+    $node->set('oe_sc_event_dates', [
+      'value' => '2022-02-15T08:00:00',
+      'end_value' => '2022-02-22T18:00:00',
+    ]);
+    $node->save();
+
+    $this->drupalGet($node->toUrl());
+    $crawler = $client->getCrawler();
+
+    $date = $crawler->filter('dl > dd');
+    $this->assertEquals('Tuesday 15 February 2022, 09.00 (CET) - Tuesday 22 February 2022, 19.00 (CET)', trim($date->text()));
```

Comment:
We would not need the trim if we would use spaceless or `-%}` syntax in the template.
The output currently has `<dd> date </dd>` instead of `<dd>date</dd>`.

Reply:
This said: The spaces are not visible in a `<dd>` element, so it's not a real problem.

Reply:
Well currently without the trim the test would fail, because of leftover space from the template.
if you inspect the output in the browser you will see leftover space within the `<dd>`.

We can ignore it and keep the `trim()` here for suppression, or we can use the space eater syntax (`-%}` instead of just `%}` in the template.

```twig
  {% set _event_dates -%}
    {{ content.oe_sc_event_dates|field_value|safe_join(', ') }}
  {%- endset %}
```

Reply:
Ticket is approaching 200% time spent, let's focus on important stuff.